src/components/PostCard/PostCard.tsx

```diff
@@ -18,8 +18,8 @@ const PostCard = ({ post }: PostCardProps): React.ReactElement => {
           className="post__image"
           src={post.imageUrl}
           alt={post.alternativeText}
-          width="150"
-          height="150"
+          width="200"
```

Comment:
¿Por qué este cambio de 150 a 200?

Reply:
necesitaban esos 50 px

Reply:
¿Para qué?

Reply:
para que cargue más grande

Reply:
Eso lo controláis por CSS.

Reply:
¿Para qué sirven los atributos width y height de la img?

Reply:
para que le reserve ese espacio